tex/chProofs.tex

```diff
@@ -1516,17 +1582,18 @@ \subsection{Using lemmas in proofs}
 Lemma dvdn_addr m d n : d %| m -> (d %| m + n) = (d %| n).
 \end{coq}
 This is a conditional equivalence, expressed as a conditional identity.
-We can replace our current goal with \C{\~\~ (p \%| 1)} by rewriting (the
+We can replace our current goal with \C{\~\~ (p \%| 1)} by rewriting
```

Comment:
Changed this to "by rewriting it using". Basically, as far as I can tell, "rewrite x" means "rewrite goal using x", not "rewrite x itself", and I'm trying to make the language reflect this.